server/src/main/java/org/candlepin/model/CdnCurator.java

```diff
@@ -46,7 +46,7 @@ public Cdn getByLabel(String label) {
                 .uniqueResult();
         }
         catch (NonUniqueResultException e) {
-            throw new IllegalStateException("Multiple CDN instances found with the same label: " + label);
+            throw new IllegalStateException("Multiple CDN instances found with the same label: " + label, e);
```

Comment:
Is this going to provide any additional details that will help debug it? I feel like it'll just add another chunk to the stack trace that we'll ignore.

Reply:
I just added it everywhere where it missing. I agree that in this specific case it's not adding a lot of value, but I think it's generally better to just have them.